index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">  
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header class="navbar">
+        <a href="#" class="nav-logo">My Logo</a>
+        <nav>
+            <ul>
+                <li><label id="checkbox_toggle" class="hamburger">&#9776;</label></li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <section class="content-section">
+            <div class="content">
+                <h1 class="content-text">I'm Andrea Glad to see you!</h1>
+                <p class="content-text-p">I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+                <p class="social-connect">LET'S CONNECT</p>
```

Comment:
- [ ] as the previous reviewer requested, kindly change the `LET'S CONNECT` to a link link tag `<a>` or a `<button>`